index.html

```diff
@@ -0,0 +1,301 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width,initial-scale=1.0">
+        <title>Website Design Project</title>
+        <link rel="stylesheet" href="css/style.css" type="text/css">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+
+    </head>
+    <body>
+        <header class="imagecontainer">
+            <div class="defineimage"></div>
+            <img src="images/ic_menu.svg" alt="menu" onclick="mobile_menu()">
```

Comment:
the x sign for closing the menu is not visible 
### Original design
![right](https://user-images.githubusercontent.com/58376020/127936336-722d11a1-b6bf-410c-867f-07ce04ea3423.png)

### Your Project
![wrong](https://user-images.githubusercontent.com/58376020/127936357-94c00ab4-fc8f-4578-ae8d-2c26954b9ba8.png)

Reply:
I have made the changes as per your suggestion. Please view my code using responsive 400 in mobile view.